core/templates/dev/head/library/Library.js

```diff
@@ -106,6 +106,14 @@ oppia.controller('Library', [
       }
     };
 
+    $scope.highlightGroup = function(groupIndex) {
+      $scope.currentGroupIndex = groupIndex;
```

Comment:
At the top of this controller, can we declare `$scope.activeGroupIndex = null;`? On reflection, I think 'active' is a better descriptor than 'current'.

And then maybe we can call the functions "setActiveGroup" and "clearActiveGroup".